modules/stereo/include/opencv2/stereo.hpp

```diff
@@ -47,8 +47,8 @@
 #include "opencv2/core.hpp"
 #include "opencv2/features2d.hpp"
 #include "opencv2/core/affine.hpp"
-#include "../../stereo/src/descriptor.hpp"
-#include "../../stereo/src/matching.hpp"
+#include "stereo/descriptor.hpp"
```

Comment:
could you then replace "stereo/descriptor.hpp" with "opencv2/stereo/descriptor.hpp" and do the same with subsequent include too for better clarity?